package.json

```diff
@@ -2,7 +2,7 @@
   "name": "api.qunitjs.com",
   "title": "QUnit API Docs",
   "description": "API reference documentation for QUnit",
-  "version": "1.17.1",
+  "version": "1.17.2",
```

Comment:
We do version bumps on the websites through `npm version patch` to also create a matching tag and pushing the tag deploys the website to production. So we don't  bump version numbers in PR's